source/documentation/using_ci/index.md

```diff
@@ -0,0 +1,13 @@
+# Configuring a custom continuous integration (CI) system
+
+Setting up a CI service lets you make sure your code changes are delivering new features without introducing new bugs or breaking existing functionality. This helps you to iterate code faster and with greater confidence, a huge benefit once the codebase becomes larger and more complex. 
+ 
+For advanced projects that need high levels of control over configuration, a tool like [Jenkins](https://jenkins.io/) is often used. For simpler projects, or where a team needs to manage deployments of prototypes from their github repositories rather than local code, cloud-based services such as [Travis CI](https://travis-ci.com/) or [CircleCI](https://circleci.com/) are common. Instructions to configure both Jenkins and Travis to run on GOV.UK PaaS are below.
+
+Security is a key concern when setting up a CI system that interacts with a PaaS. You should protect confidential data by: 
+- running your tests in an isolated environment
```

Comment:
I think it needs to be more specific. Isolating environments for tests is good advice for non-security reasons, but if we feel that it's relevant for *us* to mention in relation to security then we should say why.

Are we recommending something like the following?

- do run your CI tests and then deploy to Cloud Foundry
- do not run your CI tests against Cloud Foundry